examples/AI-surrogate-mode-hard-magnets.ipynb

```diff
@@ -0,0 +1,384 @@
+      "text/plain": [
+       "<Figure size 1440x480 with 3 Axes>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "height": 448,
+       "width": 1187
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "results_kuzmin.plot();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f7867930",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## AI-based prediction of extrinsic properties\n",
+    "Rather than running micromagnetic simulations to compute hysteresis loops and derive extrinsic properties such as coercive field Hc and maximum energy product BHmax, we use pre-trained machine learning models from the `mammos-ai` package.\n",
+    "\n",
+    "Details about the models and their training data can be found in the [mammos-ai documentation](https://mammos-project.github.io/mammos/api/mammos_ai.html).\n",
+    "\n",
+    "This approach allows for rapid predictions of hysteresis properties based on the intrinsic micromagnetic parameters computed earlier.\n",
+    "\n",
+    "The model has been trained on a 50 x 50 x 50 nm cubic grain with uniaxial anisotropy and applied field applied along the cube's edge."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "4e143880",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "T = np.linspace(0, 1.1 * results_kuzmin.Tc.q, 20)\n",
+    "results_ml = mammos_ai.Hc_Mr_BHmax_from_Ms_A_K(\n",
+    "    Ms=results_kuzmin.Ms(T),\n",
+    "    A=results_kuzmin.A(T),\n",
+    "    Ku=results_kuzmin.K1(T),\n",
```

Comment:
We pass `K1` to `Ku` and the function name contains just `K`. This all looks inconsistent. Could we homogenize it?